pkg/agent/agent_windows.go

```diff
@@ -73,6 +74,14 @@ func (i *Initializer) prepareHostNetwork() error {
 		return err
 	}
 	i.nodeConfig.UplinkNetConfig.DNSServers = dnsServers
+	// Save routes which are configured on uplink interface.
```

Comment:
on the uplink interface
configuration of the uplink interface to the OVS bridge interface.

Reply:
Done